content/en/guide/v10/hooks.md

```diff
@@ -7,12 +7,11 @@ description: 'Hooks in Preact allow you to compose behaviours together and re-us
 
 Hooks is a new concept that allows you to compose state and side effects. They allow you to reuse stateful logic between components.
 
-If you've worked with Preact for a while you may be familiar with patterns like `render-props` and `higher-order-components` that try to solve the same. But they've always made code harder to follow whereas with hooks you can neatly extract that logic and make it easy to unit test it independently.
+If you've worked with Preact for a while, you may be familiar with patterns like "render props" and "higher order components" that try to solve these challenges. These solutions have tended to make code harder to follow and more abstract. The hooks API makes it possible to neatly extract the logic for state and side effects, and also simplifies unit testing that logic independently from the components that rely on it.
 
-Hooks can only be used in function components and avoid many pitfalls of the `this` keyword that's present in classes. Instead they rely on closures which makes them value-bound and eliminates a whole bag of bugs when it comes to async state updates.
+Hooks can be used in any component, and avoid many pitfalls of the `this` keyword relied on by the class components API. Instead of accessing properties from the component instance, hooks rely on closures. This makes them value-bound and eliminates a number of stale data problems that can occur when dealing with asynchronous state updates.
```

Comment:
Does preact allow for `useState()` inside of class components, too?

Reply:
yup

Reply:
cool, I didn't know that. 👍